### DISCUSSION TITLE
Are Groups are being populated correctly from source re LiveTV/Movies/Series

### DISCUSSION THREAD

**Discussion by jamesarbrown** [euzu/tuliprox#77] [General]
Hi if I do a 1:1 copy of an upstream source and in that source there are two groups "Comedy", one for Movies and one for "Series", it seems to me that m3u is putting it all into one Group?

I think thats is what is happening, just not 100% sure

**Comment by euzu** [comment]
@jamesarbrown if you filter channels and put them into a group with the name "Comedy" everything matching wil be put in it. I need to think about this problem.

**Comment by euzu** [comment]
@jamesbrown yout problem should be solved with the  master branch. There are now  Type filter.

**Comment by euzu** [comment]
closed due inactivity
